### PR TITLE
Add Gymnasium v1.0 blog post

### PR DESCRIPTION
**Pull request by pseudo-rnd-thoughts**

### CODE REVIEW

File: _posts/2024-10-01-Gymnasium-v1.0.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+```python
+envs = gym.make_vec("CartPole-v1", num_envs=3, vectorization_mode="vector_entry_point")
+```
+
+With this, we also support custom vectorization, defined for each environment separately, which allows for significantly more efficient implementations in certain cases.
+
+We also changed the autoreset behavior of vectorized environments. In the past, when an episode terminated (or was truncated), it would immediately reset, with the final observation being passed in the `info` dict. This was less than ideal, as `info` was primarily intended for auxiliary information, not the main observations. Now, the observation returned alongside `terminated | truncated == True` is the final observation of the episode that just finished. The action following that will be ignored and treated as a reset signal, after which the new episode will begin. We will further elaborate on this in a future blog post.
```
Comment (Kallinteris-Andreas) [inline _posts/2024-10-01-Gymnasium-v1.0.md:35]: "auto-reset behavior"

File: _posts/2024-10-01-Gymnasium-v1.0.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## 2. **Improved Wrappers**
+With the separation of `Env` and `VectorEnv`, wrappers have also undergone changes. Wrappers that previously worked for both environments now have distinct variants for each. Standard environment wrappers are housed in `gymnasium.wrappers`, while vector-specific wrappers are found in `gymnasium.wrappers.vector`.
+
+Several wrappers have been renamed or removed for clarity, such as: `AutoResetWrapper` -> `Autoreset` and `FrameStack` -> `FrameStackObservation`.
+New wrappers have also been introduced, such as `DelayObservation` and `MaxAndSkipObservation`, to extend functionality.
+
+### 3. **Functional Environments**
+To enable a different coding paradigm and allow for easier integration into planning algorithms, Gymnasium introduces `FuncEnv`, a functional version of `Env`. This new environment type exposes functions such as `reward`, `observation`, and `transition`, allowing for more flexible control over the environment's behavior. Furthermore, functional environments implemented in Jax can often be jitted and vmapped, leading to masive performance improvements and easy hardware acceleration.
```
Comment (Kallinteris-Andreas) [inline _posts/2024-10-01-Gymnasium-v1.0.md:44]: 1) "massive"
2) "JAX"
3) "JITed"

File: _posts/2024-10-01-Gymnasium-v1.0.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+This problem of having Gymnasium depend on a fork of a fork of the ALE meant that any changes (such as new environments) or bug fixes that occurred in the ALE were not available in Atari-Py, which included the benchmark for 56 Atari environments that DeepMind first used in their work showing human level performance in the environments. This was made worse by the fact that DeepMind had made their own major modifications and upgrades to the ALE in this time period (some were published publicly in [Xitari](https://github.com/google-deepmind/xitari), some were not). This created well founded scientific reproducibility concerns from the original ALE authors, which led to agreements between the original ALE authors, DeepMind and OpenAI to upstream python bindings from Atari-Py and ALE. Then, Gym would depend directly on the ALE, and DeepMind would contribute all their code to ALE so that the public could use it in a centralized and reproducible way. With the release of Gymnasium 1.0, we're finally completing this process of upstreaming all Atari environments from Gymnasium to ALE.
```
Comment (Kallinteris-Andreas) [inline _posts/2024-10-01-Gymnasium-v1.0.md:70]: "well-founded"

**Review by Kallinteris-Andreas — CHANGES_REQUESTED** [review]
Overall looks good

File: _posts/2024-10-01-Gymnasium-v1.0.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+layout: blog
+short_title: "Gymnasium v1.0"
+subtitle: "Gymansium v1.0: A Comprehensive Update, Summarising the Changes"
+title: "Gymnasium v1.0: A Comprehensive Update"
+date: "2024-10-01"
+excerpt: ""
```
[the remark this answers is not in the document]
Comment (pseudo-rnd-thoughts) [reply _posts/2024-10-01-Gymnasium-v1.0.md:7]: Thanks, added

File: _posts/2024-10-01-Gymnasium-v1.0.md (hunk 2 above)
Comment (pseudo-rnd-thoughts) [reply _posts/2024-10-01-Gymnasium-v1.0.md:44]: Thanks, I'm going to keep `jitted` as this is often how is referred in the docs

**Pull request merged**